### PR TITLE
ci(cla-check): add CLA check for ubuntu-20.04

### PR DESCRIPTION
**Pull request by cjdcordeiro**
Needs #120 

This PR adds a new job that will call the CLA check reusable workflow (from https://github.com/canonical/chisel-releases/pull/131) on every PR.

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+name: CI
+run-name: CI for ${{ github.ref }}
+
+on:
+  push:
```
Comment (linostar) [inline .github/workflows/ci.yaml:5]: What's the use of `on push` here if the jobs only works for pull_request events?

**Review by linostar — CHANGES_REQUESTED** [review]
What

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (cjdcordeiro) [reply .github/workflows/ci.yaml:5]: Cause this will be merged with the work being done in #120 . It should all be part of the same workflow

**Review by linostar — APPROVED** [review]
LGTM

**Review by rebornplusplus — APPROVED** [review]
Looks good to me, thanks.

But let's block this until #120 and #131 are in.

**Pull request merged**